org.messerlab.slimgui.yml

```diff
@@ -0,0 +1,47 @@
+app-id: org.messerlab.slimgui
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+  - --talk-name=com.canonical.AppMenu.Registrar # required for global menu
+  - --talk-name=org.freedesktop.Flatpak # allows spawning processes on the host via flatpak-spawn --host
+  - --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig
```

Comment:
why do you need this at runtime?

Reply:
`- --talk-name=org.freedesktop.Flatpak` is needed, as SLiMgui is a modeeling environment with a scripting language which has the ability to call a `system()` function to spawn other processes. Some evolutionary models may need to call `R` or `python` during execution, and so the `slimgui` process needs to be able to spawn other processes.

Reply:
It has been some time since I wrote the original manifest, more than a year (May 2021). I do not recall why I included `- --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig`, but I will consult the documentation and previous versions to eek out why I might have included this permission.

Reply:
`- --filesystem=home` is needed because, as an IDE, the user may wish to use files in various places and not simply restricted to `~/Downloads` or `~/Pictures`. `~/home` should be writable, a user may wish to save their model output to `~/Documents/output/`, while writing their model scripts to `~/code/SLiM/model_3/model.slim`.

Reply:
`- --talk-name=com.canonical.AppMenu.Registrar # required for global menu` I believe at the time I wrote the original manifest we supported Ubuntu 17.04, which has Unity, but it is still possible that some users of our application are using global application menus on KDE systems. I don't know if this is still necessary.

We only support Qt 5.9.5 and later, which means we only support Ubuntu 18.04 LTS and later and these do not have the Unity desktop.

Do you have advice for me? Should I remove it?

Reply:
so I quoted only one line and didn't get a response about it

Reply:
Ah, I didn't see it was limited to line thirteen. I'll do some digging and figure out if it's necessary or not. It probably isn't.

Reply:
As far as I can tell, the permission is indeed unnecessary. I removed it.